assets/languages.txt

```diff
@@ -0,0 +1,6 @@
+zh-CN;chinese
+ar-AR;arabic
+jp-JP;japan
+ko-SK;korean
+ko-NK;korean
+bg-BG;cyrillic
```

Comment:
cyrillic is not the name of a language. What you have here is Bulgarian. Also, you should write the name of the languages in their native language, so that when someone who barely speaks english is fumbling their way through the setting they can immediately see which is their native language